isb_web/isb_solr_query.py

```diff
@@ -628,6 +628,28 @@ def solr_records_forh3_counts(
     return response.json()
 
 
```

Comment:
Ask solr to facet by the `source` column, and parse out the response to a `dict` that's easy for callers to use.